### PR TITLE
Finished coding section location about page

### PR DESCRIPTION
**Pull request by dovecancode**

### CODE REVIEW

**Comment by mrddove** [comment]
I forgot to attach the screenshot. I will do it here instead.
I have a container div centering the content in the viewport. It only sets to the section I created but later on let's set it out to a global-style container so that everyone can use it. below is the code
.location .container {
    width: 100%;
    max-width: 1200px;
    margin-left: auto;
    margin-right: auto;
    padding-left: 3%;
    padding-right: 3%;
}
image

File: about.html [hunk 1]
```diff
@@ -3,7 +3,27 @@
...
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>About | Arroceros Urban Forest Park</title>
+    <link rel="stylesheet" href="css/index.css" />
   </head>
-  <body></body>
+  <body>
+    <section class="location">
+      <div class="container">
```
Comment (melmarieee) [inline about.html]: i think we should use the container fluid so that our output will be the same.

Comment (mrddove) [reply about.html]: I did not use bootstrap its all hard-coded. container-fluid is bootstrap right? what do you suggest?

**Comment by melmarieee** [comment]
please make your description imperative something like "add  coding section location about page" and please update the pr description for your output description and screenshot, thanks.

File: about.html (hunk 1 above)
Comment (melmarieee) [reply about.html]: lets not use hard coded, lets use the bootstrap container. WDYT?

**Comment by mrddove** [comment]
I recoded it now to bootstrap

**Review by melmarieee — APPROVED** [review]
LGTM

**Pull request merged**